setup.py

```diff
@@ -1,6 +1,27 @@
 from distutils.core import setup
+import os
 import setuptools
+import sys
+
+
+python_version = sys.version_info
+if python_version[0] < 3:
+    raise RuntimeError("Python version is {}. Requires 3.6 or greater."
+                       "".format(sys.version_info))
+elif python_version[0] == 3 and python_version[1] < 6:
```

Comment:
So it is. I remembered using `StrictVersion` once but then couldn't remember why it was useful and ended up here. Comparing tuples seems like the way to go.